packages/utils/src/address.ts

```diff
@@ -7,30 +8,25 @@ import {
 } from 'ethereumjs-util'
 import * as Web3Utils from 'web3-utils'
 
-const HEX_REGEX = /^0x[0-9A-F]*$/i
-
-export type Address = string
-
-export const eqAddress = (a: Address, b: Address) => normalizeAddress(a) === normalizeAddress(b)
-
-export const normalizeAddress = (a: Address) => trimLeading0x(a).toLowerCase()
-
-export const normalizeAddressWith0x = (a: Address) => ensureLeading0x(a).toLowerCase()
-
-export const trimLeading0x = (input: string) => (input.startsWith('0x') ? input.slice(2) : input)
-
-export const ensureLeading0x = (input: string) => (input.startsWith('0x') ? input : `0x${input}`)
-
-// Turns '0xce10ce10ce10ce10ce10ce10ce10ce10ce10ce10'
-// into ['ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10','ce10']
-export const getAddressChunks = (input: string): string[] =>
-  trimLeading0x(input).match(/.{1,4}/g) || []
-
-export const isHexString = (input: string) => HEX_REGEX.test(input)
-
-export const hexToBuffer = (input: string) => Buffer.from(trimLeading0x(input), 'hex')
-
-export const bufferToHex = (buf: Buffer) => ensureLeading0x(buf.toString('hex'))
+// Exports moved to @celo/base, forwarding them
+// here for backwards compatibility
+export {
```

Comment:
you can make an `export * from @celo/base/lib/address` to export everything
Same with the others. This could probably make us avoid some problems in the future, when someone adds something in the base lib, but not in the utils

Reply:
This is exactly what I didn't want, but we can discuss if it makes sense or not. I explicitly exported just what it existed before to ensure backwards compatibility, but the idea is for people to use base, not utils, when possible. Therefore, after a while we could even delete this exports here (after carefully deprecating and searching for their usages). If a new function is added in base, I didn't want people to point to utils to use it. Does that make sense?

Reply:
makes sense